libs/vm-modules/src/math/bignumber.cpp

```diff
@@ -352,12 +315,6 @@ void UInt256Wrapper::Divide(Ptr<Object> &lhso, Ptr<Object> &rhso)
     lhs->number_ /= rhs->number_;
     return;
   }
-  if (rhs->IsTemporary())
-  {
-    rhs->number_ /= lhs->number_;
```

Comment:
Removed this block, because it division is not commutative operation => rhs & lhs can NOT be swapped